runtime/parser/declaration.go

```diff
@@ -596,7 +594,7 @@ func isNextTokenCommaOrFrom(p *parser) bool {
 	}
 }
 
-func parseHexadecimalLocation(memoryGauge common.MemoryGauge, literal string) common.AddressLocation {
+func parseHexadecimalLocation(p *parser, memoryGauge common.MemoryGauge, literal string) common.AddressLocation {
```

Comment:
Doesn't seem like we need to pass the latter two arguments to this function anymore since they are both accessible from the parser.

Reply:
Ah yes. Refactored in c42f4ae